addons/mail/static/src/js/chatter.js

```diff
@@ -430,7 +432,7 @@ var Chatter = Widget.extend({
      * @private
      */
      _updateAttachmentCounter: function () {
-        var count = this.record.data.message_attachment_count || 0;
+        var count = this.record.data.message_attachment_count || '';
```

Comment:
No need to change this LOC with option `displayCounter = false` since `.o_chatter_attachment_button_count` does not exist in the DOM.